code/__DEFINES/reagents.dm

```diff
@@ -47,41 +47,52 @@
 
 //Used in holder.dm/equlibrium.dm to set values and volume limits
 ///stops floating point errors causing issues with checking reagent amounts
-#define CHEMICAL_QUANTISATION_LEVEL 0.0001 
+#define CHEMICAL_QUANTISATION_LEVEL 0.0001
 ///The smallest amount of volume allowed - prevents tiny numbers
-#define CHEMICAL_VOLUME_MINIMUM 0.001 
+#define CHEMICAL_VOLUME_MINIMUM 0.001
 ///Round to this, to prevent extreme decimal magic and to keep reagent volumes in line with perceived values.
-#define CHEMICAL_VOLUME_ROUNDING 0.01 
+#define CHEMICAL_VOLUME_ROUNDING 0.01
 ///Default pH for reagents datum
-#define CHEMICAL_NORMAL_PH 7.000 
+#define CHEMICAL_NORMAL_PH 7.000
 
 //reagent bitflags, used for altering how they works
 ///allows on_mob_dead() if present in a dead body
-#define REAGENT_DEAD_PROCESS (1<<0) 
+#define REAGENT_DEAD_PROCESS (1<<0)
 ///Do not split the chem at all during processing - ignores all purity effects
-#define REAGENT_DONOTSPLIT (1<<1) 
+#define REAGENT_DONOTSPLIT (1<<1)
 ///Doesn't appear on handheld health analyzers.
-#define REAGENT_INVISIBLE (1<<2) 
+#define REAGENT_INVISIBLE (1<<2)
 ///When inverted, the inverted chem uses the name of the original chem
-#define REAGENT_SNEAKYNAME          (1<<3)  
+#define REAGENT_SNEAKYNAME          (1<<3)
 ///Retains initial volume of chem when splitting for purity effects
-#define REAGENT_SPLITRETAINVOL      (1<<4)  
+#define REAGENT_SPLITRETAINVOL      (1<<4)
 //Lets a given reagent be synthesized important for random reagents and things like the odysseus syringe gun(Replaces the old can_synth variable)
-#define REAGENT_CAN_BE_SYNTHESIZED  (1<<5)  
+#define REAGENT_CAN_BE_SYNTHESIZED  (1<<5)
 
 //Chemical reaction flags, for determining reaction specialties
 ///Convert into impure/pure on reaction completion
-#define REACTION_CLEAR_IMPURE       (1<<0)  
+#define REACTION_CLEAR_IMPURE       (1<<0)
```

Comment:
Remove the alignment, here and everything else you touched.

Reply:
man uses a BUSTED ass regex and dares to blame me....smh...we live in a society.

Reply:
dude asks me whos sed

Reply:
literaly who